.travis.yml

```diff
@@ -33,11 +33,14 @@ script:
   - |
     if [[ "$TRAVIS_BRANCH" == *"-stable" ]]; then
       export CTLR_VERSION=v$(echo $TRAVIS_BRANCH | sed s/-stable//g)
+      export LICENSE_STRICT="true"
```

Comment:
I recall we had a discussion on this on an email thread (or perhaps on Slack) as to when this should be enabled. If I recall, we should only enable strict checking on tagged release builds.

Reply:
I have pushed that change.